pkg/gardener/terraformer/terraformer.go

```diff
@@ -31,6 +33,31 @@ func (t *terraformer) SetVariablesEnvironment(tfVarsEnvironment map[string]strin
 	return &terraformer{t.tf.SetVariablesEnvironment(tfVarsEnvironment)}
 }
 
+// SetJobBackoffLimit implements Terraformer.
+func (t *terraformer) SetJobBackoffLimit(val int32) Interface {
```

Comment:
I think we can improve the terraform options handling in g/gardener with the usage of [functional options](https://github.com/tmrts/go-patterns/blob/master/idiom/functional-options.md).
However it is out of context of this PR.

Reply:
Yes, good idea.